lib/form-validation.js

```diff
@@ -6552,10 +6555,9 @@
 
   var _showErrors = function _showErrors(form, errors) {
     vl.util.each(form.querySelectorAll("input, textarea, select, [data-vl-error-placeholder]"), function (input) {
-      if (input.hasAttribute(fvErrorMsgAtt) && input.hasAttribute(fvErrorPlchAtt) && input.name.length && form.querySelector("[".concat(fvErrorIDAtt, "=\"").concat(input.getAttribute(fvErrorPlchAtt), "\"]"))) {
-        _showSuccessForInput(input, errors);
-
-        _showErrorsForInput(input, errors && errors[input.name]);
+      if ( input.hasAttribute(fvErrorPlchAtt) && input.name.length && form.querySelector("[".concat(fvErrorIDAtt, "=\"").concat(input.getAttribute(fvErrorPlchAtt), "\"]"))) {
```

Comment:
vergeet zeker niet eens te linten

Reply:
had ik gedaan... dacht ik. Ttz ik had op 't einde een npm install gedaan in de veronderstelling dat die mij foutmeldingen zou geven als de linting niet inorde was.

Reply:
denk niet dat npm install dit gaat controleren

Reply:
Misschien zit die in de lib folder niet in wat moet gelint worden. Ik zie echter wel een spatie teveel staan, daarmee dat ik het zei.
```suggestion
      if (input.hasAttribute(fvErrorPlchAtt) && input.name.length && form.querySelector("[".concat(fvErrorIDAtt, "=\"").concat(input.getAttribute(fvErrorPlchAtt), "\"]"))) {
```